lib/elixir/test/erlang/tokenizer_test.erl

```diff
@@ -1,8 +1,33 @@
 -module(tokenizer_test).
 -include_lib("eunit/include/eunit.hrl").
+% -import(elixir_tokenizer, [tokenize/3]).
```

Comment:
Oh well, sorry for the forgotten commented out code, I didn't know how to run an individual test file with EUnit in the beginning, so probably that's why the code was there